### PR TITLE
Part responsible owner

### PR DESCRIPTION
**Pull request by SchrodingersGat**
- Replaces https://github.com/inventree/InvenTree/pull/5060
- Fixes https://github.com/inventree/InvenTree/issues/5059

### CODE REVIEW

File: InvenTree/part/serializers.py [hunk 1]
```diff
@@ -695,6 +696,11 @@ def get_starred(self, part):
         read_only=True,
     )
 
+    responsible = serializers.PrimaryKeyRelatedField(
+        queryset=users.models.Owner.objects.all(),
+        source='responsible_owner',
+    )
+
```
Comment (matmair) [inline InvenTree/part/serializers.py:703]: Comment: This should lead to a bumb of the API version, right?

**Review by matmair — APPROVED** [review]
lgtm

**Comment by SchrodingersGat** [comment]
@miggland FYI this is how complex migrations should be handled. In separate sequential migrations:

1. Add the new field to the model
2. Perform custom data migration
3. Delete the old field

Data and schema migrations should not be performed in the same step. And it should be written in such a way that it is reversible if required :)

**Comment by miggland** [comment]
Looks nice, thanks for fixing this. :+1:

**Comment by SchrodingersGat** [comment]
No worries, it is a pretty tricky thing to get right!

**Pull request merged**